src/cloud-api-adaptor/Makefile

```diff
@@ -121,8 +121,8 @@ image-with-arch: .git-commit ## Build the per arch image
 .PHONY: deploy
 deploy: ## Deploy cloud-api-adaptor using the operator, according to install/overlays/$(CLOUD_PROVIDER)/kustomization.yaml file.
 ifneq ($(CLOUD_PROVIDER),)
-	kubectl apply -k "github.com/confidential-containers/operator/config/release?ref=v0.8.0"
-	kubectl apply -k "github.com/confidential-containers/operator/config/samples/ccruntime/peer-pods?ref=v0.8.0"
+	kubectl apply -k "github.com/confidential-containers/operator/config/overlays/peerpods/default"
+	kubectl apply -k "github.com/confidential-containers/operator/config/samples/ccruntime/peer-pods"
```

Comment:
I found some other files with `samples/ccruntime/peer-pods` reference, should we update them as well?

 * docs/Release-Process.md
 * src/cloud-api-adaptor/test/provisioner/provision.go

Reply:
Are you proposing that docs changes, test changes and code changes for related changes are all merged in the same commit?

Reply:
I don't necessarily disagree - but I've already merged more commits together than the CoCo (and Kata) guidelines suggest: https://github.com/confidential-containers/confidential-containers/blob/main/CONTRIBUTING.md#commit-format 
> The "subsystem" describes the area of the code that the change applies to. It does not have to match a particular directory name in the source tree because it is a "hint" to the reader. The subsystem is generally a single word.

> If the commit touches many different subsystems, you might want to split it into multiple commits. In general more smaller commits are preferred to fewer larger ones.

Reply:
> Are you proposing that docs changes, test changes and code changes for related changes are all merged in the same commit?

Yes, this would improve our bisect ability, prevent missed commits during reverts/cherry-picks, and make it easier to drop commits that don't make sense, since we don't have to look for related ones.

I know this could be controversial, but I also don't think you would be breaking the guidelines here:

> **It does not have to match** a particular directory name in the source tree because it is a "hint" to the reader.

> If the commit touches many different subsystems, you **might want** to split it into multiple commits.

The guideline *suggests* splitting commits that affect multiple subsystems. This is particularly relevant if each subsystem involves distinct changes. However, if the modifications are closely related or minor, it might be more practical to keep them together in a single commit. 

However, I understand you've put a lot of effort into this. So, feel free to ignore the nit ones. It would be great if we could adopt this in future PRs, tho.

Reply:
Given that you are the only reviewer, I've updated the changes to match what you would like.

Reply:
Thanks @stevenhorsman, and sorry if I wasn't clear before, there was no *real* need for that, But appreciated.